pages/index.jsx

```diff
@@ -65,12 +64,24 @@ function Home() {
     );
   });
 
-  const eventCards = EVENT_DATA.map((event) => (
+  const [eventData, setEventData] = useState([]);
+  const [loadingEvents, setLoadingEvents] = useState(true);
+
+  useEffect(() => {
+    if (loadingEvents) {
+      setLoadingEvents(false);
+      fetch('https://events.uf-ace.com/api/events/')
+        .then((response) => response.json())
+        .then(({ content }) => setEventData(content));
+    }
+  });
+
+  const eventCards = eventData.map((event) => (
     <Event
       key={event.summary}
-      name={event.summary}
-      description={event.description}
-      datetime={event.datetime}
+      name={event.name}
+      description={event.description.replace(/<\/?\w+>/g, '').replace('&nbsp;', '').slice(0, 250)}
```

Comment:
No. Just no.

Reply:
Fair, will need to find a package to strip it more seemlessly

Reply:
@ChristopherJMiller that's not my worry, my worry is why are we doing this in the first place?

Reply:
Resolved with most recent commit, changed to using short_description